h2o-algos/src/main/java/hex/tree/drf/DRFModel.java

```diff
@@ -73,4 +77,17 @@ public DRFParameters() {
     }
   }
 
+  @Override
+  public Model<DRFModel, DRFModel.DRFParameters, DRFModel.DRFOutput>.RawDataStreamWriter getRawDataStream() {
+    return new RawDataStreamWriter();
+  }
+
+  public class RawDataStreamWriter
+          extends SharedTreeModel<DRFModel, DRFModel.DRFParameters, DRFModel.DRFOutput>.RawDataStreamWriter {
+    @Override
+    protected void writeExtraModelInfo() throws IOException {
+      super.writeExtraModelInfo();
```

Comment:
Hmm, this seemed like an obvious pattern to me: there is a base RawDataStreamWriter class that implements base functionality, and then each model subclass augments that functionality by extending the base class... The `getRawDataStream` function is just a helper to instantiate the correct class